app/src/main/res/values/array.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string-array name="verification_methods">
+        <item>Visual verification</item>
+        <item>Rhythm verification</item>
+        <item>Choose when pairing</item>
+    </string-array>
```

Comment:
> I think it's needed to have this file if we would like to expand it with more static arrays.

In that case, I would suggest you do the same for [the dutch version of `strings.xml`](https://github.com/ericcornelissen/NervousFish/pull/214/files#diff-b0d05aebee027c15c77e36003f7bcadcR108)


> The name 'array.xml' is the default name for arrays, what would you give as a different name?

Seems a bit odd to me given .e.g string**s**.xml, but how does array**s**.xml sound?